src/components/drawer-content/drawer-content.tsx

```diff
@@ -173,7 +173,7 @@ const DrawerContent: React.FC<Props> = () => {
               paddingRight={ 3 }
               width="100%" 
             >
-              { renderAccordinRow(`${strings.total}:`, totalHour, totalColor) }
+              { renderAccordinRow(`${strings.difference}:`, totalHour, totalColor) }
```

Comment:
It can just use total
```suggestion
              { renderAccordinRow(`${strings.total}:`, totalHour, totalColor) }
```